pkg/controller/actions/actions.go

```diff
@@ -31,8 +44,10 @@ func CreateOrUpdateOwnedResource(ctx *core.Context, owner metav1.Object, control
 		return nil
 	}
 
-	ctx.Log.V(1).Info("Applying patch to object", "gvk", gvk, "object", controlled, "patch", string(patchResult.Patch))
-	if err = ctx.Patch.Annotator.SetLastAppliedAnnotation(controlled); err != nil {
+	ctx.Log.V(1).Info("Applying patch to object", "gvk", gvk, "object", controlled, "patch",
+		string(patchResult.Patch))
+	err = ctx.Patch.Annotator.SetLastAppliedAnnotation(controlled)
```

Comment:
Not sure

Reply:
This isnt clear to me either, shouldnt kube-apiserver populate this field. TBH I dont understand the new apply mechanism that this field relates to :(. 

@sonnysideup ?

Reply:
Never mind, then. I will separately look at what's going on there. This entire piece of code will need some final refactoring, anyway.